lib/livebook/storage/ets.ex

```diff
@@ -98,13 +107,38 @@ defmodule Livebook.Storage.Ets do
 
     :ets.insert(table, attributes)
 
+    :ok = save_to_file(state)
+
     {:reply, :ok, state}
   end
 
   @impl GenServer
   def handle_call({:delete, namespace, entity_id}, _from, %{table: table} = state) do
     :ets.delete(table, {namespace, entity_id})
 
+    :ok = save_to_file(state)
+
     {:reply, :ok, state}
   end
+
+  defp table_name(), do: :persistent_term.get(__MODULE__)
+
+  defp load_or_create_table(file_path) do
+    file_path
```

Comment:
Just for consistency with the counterpart below :)

```suggestion
  defp load_or_create_table() do
    config_file_path()
```